armada/handlers/armada.py

```diff
@@ -82,11 +82,11 @@ def pre_flight_checks(self):
             if ct_type == 'git' and not git.check_available_repo(location):
                 raise ValueError(str("Invalid Url Path: " + location))
 
-            if not self.tiller.tiller_status():
```

Comment:
the pre_flight_checks method iterates through each chart and these are currently inside the loop